tests/server/window_report_server_test.py

```
@@ -0,0 +1,125 @@
+        # Reach past the client helper and send an arbitrary other method — it
+        # must be rejected. This is the security boundary: no device control.
+        c = self._client()
+        p.send_message(c._conn, p.make_request(99, p.M_BRIGHTNESS_SET, {"value": 50}))
+        self.assertTrue(c._conn.poll(3.0))
+        msg = p.recv_message(c._conn)
+        self.assertIn("error", msg)
+        self.assertEqual(msg["error"]["code"], p.ERR_METHOD_NOT_FOUND)
+        self.assertEqual(self.reports, [])  # callback never ran
+
+    def test_wrong_authkey_cannot_connect(self):
+        with self.assertRaises(Exception):
```

Ruff (0.15.15)

[warning] Do not assert blind exception: `Exception`

(B017)